### DISCUSSION TITLE
SOC Product Brief

### DISCUSSION THREAD

**Discussion by Hello-dot-Jpg** [nk64/GT4286#11] [Show and tell]
https://comake.online/uploadfile/file/20220406/20220406022923_16431.pdf

**Comment by horsiezoo** [comment]
Fantastic, pins 34 and 35 uart seem important

**Reply by Hello-dot-Jpg** [reply]
What's really interesting is that it has built in ethernet?!

**Reply by nk64** [reply]
If you can get an ethernet dongle plugged in, adbd is [running](https://github.com/nk64/GT4286/tree/main/sdcard_tweaks/roms/MAME/output/ps.txt) on it and maybe we can get access to a real-time shell and logging.
